frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboard.tsx

```diff
@@ -1,12 +1,43 @@
 import * as React from 'react';
+import * as _ from 'lodash';
+import { match as RMatch } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
+import { Grid, GridItem } from '@patternfly/react-core';
+import MonitoringDashboardGraph from './MonitoringDashboardGraph';
+import { queries } from './monitoringDashboardQueries';
+import MonitoringDasboardCountBlock from './MonitoringDashboardCountBlock';
+
+export interface MonitoringDashboardProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+
+const MonitoringDashboard: React.FC<MonitoringDashboardProps> = ({ match }) => {
+  const namespace = match.params.ns;
 
-const MonitoringDashboard: React.FC = () => {
   return (
     <>
       <Helmet>
         <title>Dashboard</title>
       </Helmet>
+      <Grid className="co-m-pane__body" gutter="md">
+        <GridItem span={4} rowSpan={1}>
+          <MonitoringDasboardCountBlock />
+        </GridItem>
+        {_.map(queries, (q, i) => (
+          <GridItem span={4} key={i}>
```

Comment:
```suggestion
          <GridItem span={4} key={title}>
```
Could we not use title for this? Two same-named items wouldn't be helpful, no?

Maybe `${chartType}-${title}` if we'd want the same stat in two different chart formats?